cookiecutter.json

```diff
@@ -2,7 +2,7 @@
     "project_name": "Name of the project",
     "project_short_description": "A short description of the project",
     "include_apigw": "y",
-    "include_xray": "y",
+    "include_lambda_powertools": "y",
```

Comment:
Same comment as above - What if we include this by default without prompting the user? They're good practices either way.